fabulaws/library/wsgiautoscale/templates/nginx.conf

```diff
@@ -47,6 +47,11 @@ server {
         proxy_read_timeout 30;
         proxy_connect_timeout 30;
 
+        # rewrite hosts with trailing dot to non-dot version
+        if ($http_host ~ "\.$" ){
+            rewrite ^(.*) $scheme://$host$1 permanent;
```

Comment:
actually according to the nginx docs, "If a replacement string starts with “http://” or “https://”, the processing stops and the redirect is returned to a client."  So removing "permanent" won't be enough.  Maybe something like

    if ($http_host ~ "(.*)\.$") { set $host $1; }

I don't know if that's even allowed, though.